packages/tangle/message_test.go

```diff
@@ -211,8 +211,10 @@ func TestMessage_UnmarshalTransaction(t *testing.T) {
 }
 
 func TestMessage_MarshalUnmarshal(t *testing.T) {
+	mockOTV := &SimpleMockOnTangleVoting{}
 	tangle := NewTestTangle()
 	defer tangle.Shutdown()
+	tangle.OTVConsensusManager = NewOTVConsensusManager(mockOTV)
```

Comment:
Can't we just set a mock `LikeReferencesFunc` instead of initializing a mock OTV?

Reply:
Done.